frontend/challedu-app/src/components/navbar.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from "react";
+import { Button , ButtonGroup  }  from 'reactstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
I don't think this needs to be imported more than once

Reply:
solved